airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -102,14 +111,20 @@ const DbtJobsList = ({
   isValid: boolean;
   dirty: boolean;
 }) => (
-  <div className={classNames(styles.jobListContainer, styles.emptyListContent)}>
-    <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
+  <div className={classNames(styles.jobListContainer)}>
     {jobs.length ? (
-      jobs.map((_j, i) => <JobsListItem key={i} jobIndex={i} removeJob={() => remove(i)} />)
+      <>
+        <Text className={styles.contextExplanation}>
+          <FormattedMessage id="connection.dbtCloudJobs.explanation" />
+        </Text>
+        {jobs.map((_j, i) => (
```

Comment:
Removing the unused `_j` binding was mentioned in review, but since this is a function argument list, it's a syntax error to have no binding in that position.

Reply:
Sorry oversaw that. Could we instead rename it just `job` in this case? Since we don't use the `_` notation for those, and that way it would be a better descriptive (not used) variable.

Reply:
Yep, the underscore is needed as a cue for the linter. While my default preference was to use a name which communicates the value, that's superfluous here (by `jobs.map`, a reader ought to have a good sense of what's going on :smile:) and a quick grep (I ran `rg '\b_\b'` in `airbyte-webapp/`) shows that using `_` for unused arguments is already a somewhat-established convention. It helps that we only import lodash functions individually, so there's no risk of shadowing needed utility functions.

Reply:
Sorry did not notice that we have that checked via `TS` and not eslint. We could (not as part of this PR!) maybe disable the TS check and replace it by the eslint rule, which actually allows to configure an `after-used` so it's not failing on parameters that need to be there because we use the ones after it still.